docker-compose.yml

```diff
@@ -44,8 +44,20 @@ services:
     ports:
       - 8889:8889
     restart: always
+  redis:
+    image: redis:latest
+    command: ["redis-server", "--requirepass", "mysecretpassword"]
```

Comment:
this needs to be the same as the one defined in the client code. Its better to configure them as env vars